templates/image-builder.yml

```
@@ -199,6 +199,15 @@ objects:
           left outer join blueprint_versions v on c.blueprint_version_id = v.id
           left outer join blueprints b ON v.blueprint_id = b.id
           cross join lateral jsonb_array_elements(c.request->'image_requests') as req;
+    - prefix: ${FLOORIST_QUERY_PREFIX}/blueprints
+      query: >-
+        select
+          v.blueprint_id::text as blueprint_id, v.version, b.deleted
```

```suggestion
          v.blueprint_id::text as blueprint_id, v.version, b.deleted,
```

I haven’t tried the query but I would expect a comma here?